src/averageAges.js

```diff
@@ -20,6 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = (Number.isInteger(century))
+    ? people.filter(person =>
+      (person.sex === 'm'
+      && Math.ceil(person.died / 100) === century))
+    : people.filter(person =>
+      person.sex === 'm');
```

Comment:
In both cases you used `filter`, move it outside ternary operator (just select needed condition inside filter method)